apiserver/facades/client/applicationoffers/base.go

```diff
@@ -287,11 +285,11 @@ func (api *BaseAPI) getModelsFromOffers(user names.UserTag, offerURLs ...string)
 		if ownerName == "" {
 			ownerName = user.Id()
 		}
-		model, absModelPath, ok, err := api.modelForName(url.ModelName, ownerName)
+		model, absModelPath, err := api.modelForName(ctx, url.ModelName, ownerName)
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
-		if !ok {
+		if model == nil {
```

Comment:
Based on comments above I think this should be transforming a `modelerrors.NotFound` to a `coreerrors.NotFound`

Reply:
msged you on this for clarification :)